bin/init.sh

```diff
@@ -10,10 +10,6 @@ rm -f django.db
 
 python manage.py migrate
```

Comment:
Maybe in a follow-up we parameterize this call with an environment variable, and that's how we can configure different environments?

Reply:
See https://docs.djangoproject.com/en/4.0/ref/django-admin/#migrate

Reply:
Sounds good as a follow-up!